docs/Examples.md

```diff
@@ -408,11 +408,17 @@ for more details.
 
 *Note that this requires using the Splunk Enterprise container version 8.1.0 or later*
 
+With the latest release of the Splunk Operator, an App Framework is available to centrally store and deploy apps. See [AppFramework](AppFramework.md) for information and examples.
```

Comment:
Can we say "a Beta version of the new App framework"

Also, add something like: 
The below method of installing apps continues to be supported, but will be deprecated in future releases.